tools/docker/start.sh

```diff
@@ -1,14 +1,24 @@
 #!/bin/bash
+set -ex
 
-cd web-platform-tests
-git pull --depth=50
+REPO=${1:-origin}
+BRANCH=${2:-master}
+REV=${3:-master}
+BROWSER=${4:-all}
+
+cd ~/web-platform-tests
+git fetch ${REPO} ${BRANCH} -n -q --depth=50
+git checkout -b build ${REV}
 
 sudo sh -c './wpt make-hosts-file >> /etc/hosts'
 
-# Install Chome dev
-deb_archive=google-chrome-unstable_current_amd64.deb
-wget https://dl.google.com/linux/direct/$deb_archive
+if [[ $BROWSER == "chrome"* ]] || [[ "$BROWSER" == all ]]
```

Comment:
Why the asterix after "chrome"?

Reply:
It makes it a prefix match. Otherwise `chrome-dev` isn't going to match :)

Reply:
OK, but if "chrome" is given we'll still use Chrome Dev... I guess this doesn't matter unless we want to also run something other than Chrome Dev though.